src/data/getDataBy.ts

```diff
@@ -1,5 +1,30 @@
 import {ensureArray} from 'utilities'
 
+interface ActionComboData {
```

Comment:
For consistency with existing data, these interfaces shouldn't have the `Data` suffix.

---

Esp. given precendent (thanks!) with the `AoE.d.ts`, could these be moved into the action and status folders so as to passively type the values? I've not actually tested to see if it'd work.